File: solutions/java8/easy/java-output-formatting/solution-approach1.java
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/java-output-formatting/problem?isFullScreen=true
// Problem     Java Output Formatting
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    java8
// Status      Accepted
// Submitted   2026-08-06, 11:38 a.m.
// ──────────────────────────────────────────────────

import java.util.Scanner;

public class Solution {

    public static void main(String[] args) {
            Scanner sc=new Scanner(System.in);
            System.out.println("================================");
            for(int i=0;i<3;i++)
            {
                String s1=sc.next();
                int x=sc.nextInt();
                
                System.out.print(s1);
                for(int j=s1.length();j<15;j++){
                    System.out.print(" ");
                }
                if(x<100 && x>0){
                    System.out.print("0");
                }
                if(x==0){
                    System.out.print("00");
                }
                System.out.print(x);
                System.out.println();

                
            }
            
            System.out.println("================================");

    }
}

File: solutions/cpp/easy/c-tutorial-functions/solution-approach1.cpp
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/c-tutorial-functions/problem?isFullScreen=true
// Problem     Functions
// Difficulty  Easy
// Subdomain   Introduction
// Platform    HackerRank
// Language    cpp
// Status      Accepted
// Submitted   2026-08-12, 09:36 a.m.
// ──────────────────────────────────────────────────

#include <iostream>
#include <cstdio>
#include <algorithm> // Required for std::max
using namespace std;

int max_of_four(int a, int b, int c, int d) {
    return max({a, b, c, d});
}

int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int ans = max_of_four(a, b, c, d);
    printf("%d", ans);
    return 0;
}
